### PR TITLE
Add homepage and option to choose application. Closes #4

### LINKED ISSUE #4: As an application submitter, I want to choose the application I want to fill out.
**User story:**
As an application submitter, I want to choose the application I want to fill out so that I can be confident I'm filling out the right one (and so that my application will be more likely to succeed).

**Outstanding questions:**
- Possible application types are `special use` and `Christmas tree` — are there others?

### PR DESCRIPTION
**Pull request by lauraGgit**
Would love to have you take a look at this @colinpmacarthur.

### CODE REVIEW

File: forestserviceprototype/specialuseform/templates/specialuseform/home.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+{% block content %}
+<h1>Welcome to the Forest Service Online Permits System</h1>
+
+<p>Are you an commerical outfitter?</p>
+<small>Coming Soon</small>
+<div class="button_wrapper">
+   <button class="usa-button-disabled" disabled>Yes</button>
+ </div>
+
+<p>Are you looking to run an event in the forest?</p>
```
Comment (colinpmacarthur) [inline forestserviceprototype/specialuseform/templates/specialuseform/home.html]: I think we probably want this to be something like: 

**Are you organizing a _non-commercial_ activity that involves a group of 75 or more people?** (either as participants or spectators). Noncommercial is any use or activity where an entry or participation fee is not charged, and the primary purpose is not the sale of a goods or service. For example: weddings, church services, endurance rides, regattas, camping trips, hikes, music festivals, rallies, graduations, and races.

From: http://www.fs.fed.us/specialuses/special_non_com_uses.shtml#sp-noncom-a

File: forestserviceprototype/specialuseform/templates/specialuseform/home.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% extends "specialuseform/base.html" %}
+
+{% block content %}
+<h1>Welcome to the Forest Service Online Permits System</h1>
+
+<p>Are you an commerical outfitter?</p>
```
Comment (colinpmacarthur) [inline forestserviceprototype/specialuseform/templates/specialuseform/home.html]: Maybe better to put this after the non-commercial group use permit and phrase it as **"Need to apply for another kind of permit?"** (Many groups, including, but not exclusively, commercial outfitters, need to apply for special use permits.)

Comment (colinpmacarthur) [reply forestserviceprototype/specialuseform/templates/specialuseform/home.html]: http://www.fs.fed.us/specialuses/special_app_process.shtml#sp-app-b

File: forestserviceprototype/specialuseform/templates/specialuseform/home.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+<small>Coming Soon</small>
+<div class="button_wrapper">
+   <button class="usa-button-disabled" disabled>Yes</button>
+ </div>
+
+<p>Are you looking to run an event in the forest?</p>
+<p><a href="{% url 'edit' permit.permit_id %}"><button>Yes</button></a></p>
```
Comment (colinpmacarthur) [inline forestserviceprototype/specialuseform/templates/specialuseform/home.html]: I generally like buttons to specifically name the action they initiate -- maybe we could title this one "Apply for a non-commercial group permit" ?

File: forestserviceprototype/specialuseform/templates/specialuseform/home.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{% extends "specialuseform/base.html" %}
+
+{% block content %}
+<h1>Welcome to the Forest Service Online Permits System</h1>
```
Comment (colinpmacarthur) [inline forestserviceprototype/specialuseform/templates/specialuseform/home.html]: I think we might not want to pick a system title right now. (I am secretly hoping it doesn't get called "ePermits" 😄 ) What about titling the page **Apply for a permit** ?

**Review by colinpmacarthur — COMMENTED** [review]
Hey @lauraGgit! Nice work. I'm glad we're adding this page -- I think it will open an interesting conversation about where (in the user's journey) our system should start. Part of me feels like the appropriate pages of the FS website should link directly to the applicable permit (and our system shouldn't be trying to catalog and explain the permit options to them), but let's see what the users think!

**Comment by colinpmacarthur** [comment]
Ah, one other note: I wasn't able to get this running locally, so I based my review on what I saw in the Github source code diff. Let me know if I'm missing something key by reviewing this way.

**Comment by lauraGgit** [comment]
Need to change the view that the button links to to be `submit`

**Comment by colinpmacarthur** [comment]
@lauraGgit: Made these changes. I think this page is good to go.

**Pull request merged**